src/content/docs/it/getting-started.mdx

```diff
@@ -0,0 +1,108 @@
+---
+title: Per Iniziare
+description: Un'introduzione base ad Astro
+---
+import Button from '~/components/Button.astro'
+import ContributorList from '~/components/ContributorList.astro'
+import PackageManagerTabs from '~/components/tabs/PackageManagerTabs.astro'
+
+<h2>Cos’è Astro?</h2>
+
+Astro è un **web framework** **tutto in uno** pensato per costruire siti web **veloci** e **incentrati sui contenuti**. 
+
+## Funzionalità Chiave
+
+- **Isole di Componenti:** Una nuova architettura web per costruire siti web più performanti.
+- **Design dell’API orientato al server:** Rimuovi idratazioni costose dai dispositivi dei tuoi utenti.
+- **Zero JS, di default:** Nessun bagaglio di runtime JavaScript che ti rallentì.
+- **Pronto per l’Edge:** Pubblica ovunque, anche su un runtime Edge come Deno o Cloudflare.
+- **Personalizzabile:** Tailwind, MDX, e oltre 100 integrazioni tra cui scegliere.
+- **Indipendente dalla UI:** Supporta React, Preact, Svelte, Vue, Solid, Lit e altri.
+
+Leggi la nostra analisi dettagliata [Perché Astro](/it/concepts/why-astro/) per saperne di più su cosa rende Astro speciale. ✨
+
+
+## Prova Astro nel tuo browser
+
+Visita [astro.new](https://astro.new/) e scegli tra una varietà di template per cominciare. Sperimenta con una versione completa e funzionante di Astro direttamente nel tuo browser!
+
+<div style="display: flex; flex-wrap: wrap; gap: 0.5rem;">
+  <Button href="https://astro.new/basics?on=stackblitz">Apri il template base</Button>
+  <Button variant="outline" href="https://astro.new/">Vedi tutti i template →</Button>
+</div>
+
+## Inizia il tuo primo progetto
+
+Fai partire un nuovo progetto Astro in locale con `create astro`, il nostro strumento CLI!
+
+<PackageManagerTabs>
+  <Fragment slot="npm">
+  ```shell
+  # crea un nuovo progetto con npm
+  npm create astro@latest
+  ```
+  </Fragment>
+  <Fragment slot="pnpm">
+  ```shell
+  # crea un nuovo progetto con pnpm
+  pnpm create astro@latest
+  ```
+  </Fragment>
+  <Fragment slot="yarn">
+  ```shell
+  # crea un nuovo progetto con yarn
+  yarn create astro
+  ```
+  </Fragment>
+</PackageManagerTabs>
+
+La nostra [Guida all’Installazione](/it/install/auto/) ha istruzioni complete, passo per passo, per installare Astro usando il nostro strumento CLI, creare un nuovo progetto da una repository pre-esistente di Astro su GitHub, e per installare Astro manualmente.
```

Comment:
I'm probably old school, but in my head it's "il repository" 😂

But that's okay, things changed

Reply:
It's probably more accurate "il repository" given that the literal translation is "il deposito" but I've always heard it being used this way by my professors at uni too. :joy:

Idk, maybe it's derived by a more loose translation like "la raccolta" which sounds nicer.

Reply:
Do you think it needs to be changed? L'Accademia della Crusca reports that both versions are used but that "il repository" has been historically more common.

For more context:
https://accademiadellacrusca.it/it/consulenza/quale-possibile-traducente-per-repository/2859